### PR TITLE
Fix app startup and routes

### LINKED ISSUE #435: Bassa does not start anymore
Attempt to start Bassa fails with a python `ModuleNotFound`  error:

```
(env)  ✘  ~/Code/Bassa/components/core   master ●  python Main.py
Traceback (most recent call last):
  File "Main.py", line 3, in <module>
    from REST import *
ModuleNotFoundError: No module named 'REST'
```

## Expected Behavior
Bassa server should start

## Current Behavior
Fails with `ModuleNotFoundError`

## Possible Solution
Changes issued by https://github.com/scorelab/Bassa/commit/af72fc9e2258a5c129abea5e7bb2c755e22730e8 has not been synced with `Main.py`. Changing the set of imports should do the trick, I guess.

`REST.py` has been split and renamed but, `Main.py` still depends on it.

## Steps to reproduce
1. Setup Bassa dev environment
2. run `python Main.py` inside `components/core`

## Environment
- Ubuntu 17.10
- Python 3.6.4 :: Anaconda, Inc.

### PR DESCRIPTION
**Pull request by Asutosh989**
Closes #435

This fixes issues caused by broken refactoring by #400.

## Description
* Organized routes as a subpackage
* Currently route files import `Server.py` as described in [Flask docs for Larger Applications](http://flask.pocoo.org/docs/0.12/patterns/packages/)

## Related Issue
- #435

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests pass.

### CODE REVIEW

**Comment by Asutosh989** [comment]
Circular imports are something most python devs frown upon, but here it is the most logical solution. The scope of the app is big enough that it warrants different files for different kind of routes but still it's not too big to introduce (potentially complex) blueprints.

File: components/core/Server.py [hunk 1]
```diff
@@ -46,3 +42,5 @@ def serve_ui(path):
 def serve_ui1():
     return send_file(os.path.dirname(os.path.realpath(__file__))+"/ui/index.html")
 
+# Import routes
+from routes import *
```
Comment (malithsen) [inline components/core/Server.py]: why is this at the bottom?

Comment (Asutosh989) [reply components/core/Server.py]: Circular imports

We need the rest of the code defined before the routes can use them

**Comment by Asutosh989** [comment]
@malithsen I have checked in my local system. And this PR solves the issue. Please go through it and merge it.

**Comment by saisankargochhayat** [comment]
This seems good to be merged !

**Comment by malithsen** [comment]
@Asutosh989 Thank you for the PR :+1:

**Pull request merged**